CONTRIBUTING.md

```diff
@@ -1,157 +1,189 @@
+If you couldn't find a similar issue, you can submit a new one by following these steps:
+1. Click [here](https://github.com/massalabs/thyra/issues/new/choose) to go directly to the issue creation page.
+3. Select the "Bug Report" issue type.
+4. A template will be automatically populated with the required information. Please fill out the template as completely as possible.
 
-```bash
-  sudo apt update
-  sudo apt install -y build-essential libgl1-mesa-dev xorg-dev p7zip
-```
+We really appreciate your contributions in reporting issues and helping us improve MassaStation!
 
-- Install Node.js and NPM:
 
-  - Windows:
-    - Download the Node.js installer from <https://nodejs.org/en/download/>.
-    - Run the installer and follow the prompts to install Node.js and NPM.
+## Suggesting Features
 
-  - macOS:
+If you have an idea for a new feature or enhancement, we'd love to hear about it! To suggest a new feature, follow these steps:
+1. First, check if a similar feature request has already been submitted by searching the [issues](https://github.com/massalabs/thyra/issues) page. If you find a similar feature request, you can upvote it using the 👍 reaction.
 
-    - Install Homebrew by following the instructions at <https://brew.sh/>.
+If you couldn't find a similar feature request, you can submit a new one by following these steps:
 
-    - Run the following command to install Node.js and NPM:
+2. Click [here](https://github.com/massalabs/thyra/issues/new/choose) to go directly to the issue creation page.
+3. Select the "Task" issue type.
+4. A template will be automatically populated with the required information. Please fill out the template as completely as possible.
 
-        ```bash
-        brew install node
-        ```
+We really appreciate your contributions in suggesting new features and making MassaStation better!
 
-  - Ubuntu:
-    - Run the following command to update the package lists:
 
-        ```bash
-        sudo apt update
+## Reviewing Pull Requests
 
-        ```
+Reviewing pull requests is an excellent way to contribute to the project. It helps us ensure that the codebase is well-maintained and that new features and bug fixes are properly tested and documented. If you have the time and skills to review pull requests, please consider doing so.
 
-    - Run the following command to install Node.js and NPM:
 
-        ```bash
-        sudo apt install nodejs npm
-        ```
+## Contributing fixes & features
 
-- Install Go: Go is required to build and run Thyra. You can install Go by following the instructions at [https://go.dev/doc/install](https://go.dev/doc/install).
+To ensure a smooth collaboration and avoid duplication of efforts, please follow these guidelines:
 
-- Install Swagger: Thyra uses Swagger to generate code from the API documentation. You can install Swagger by running the following command:
+1. Before starting work on a new feature or bug fix, check the project's GitHub repository for existing issues.
```

Comment:
Add a note to specify that they can start looking for "good first issue" flag and tackle any of this "easy issue". WDYT?